borg/archiver.py

```diff
@@ -896,6 +906,43 @@ def run(self, args=None):
                                type=location_validator(archive=False),
                                help='repository to prune')
 
+        convert_epilog = textwrap.dedent("""
+        convert will convert an existing Attic repository to Borg in place.
+
+        it will change the magic numbers in the repository's segments
+        to match the new Borg magic numbers. the keyfiles found in
+        $ATTIC_KEYS_DIR or ~/.attic/keys/ will also be converted and
+        copied to $BORG_KEYS_DIR or ~/.borg/keys.
+
+        the cache files are *not* currently converted, which will
+        result in a much longer backup the first time. you can run
+        `borg check --repair` to rebuild those files after the
+        conversion.
+
+        the conversion can IRREMEDIABLY DAMAGE YOUR REPOSITORY! Attic
```

Comment:
maybe so. could be my french spilling into my english...

what would you propose? irrevocably? irreversebly? hopelessly? incurably? inrreparably? incorrigibly? thesaurus are fun :)

http://www.wordreference.com/thesaurus/irremediable
http://www.thesaurus.com/browse/irremediable
http://www.merriam-webster.com/thesaurus/irremediable

i think irrevocably may be more common.